src/main/kotlin/com/plus/taxiapp/domain/PaymentRepository.kt

```diff
@@ -0,0 +1,6 @@
+package com.plus.taxiapp.domain
+
+interface PaymentRepository {
+    fun saveAccount(account: Account): Account
+    fun saveCard(card: Card): Card
```

Comment:
🚫 **[ktlint]** <standard:blank-line-before-declaration> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Expected a blank line for this declaration